test-case/check-kmod-load-unload-after-playback.sh

```diff
@@ -48,7 +48,9 @@ OPT_HAS_ARG['d']=1         OPT_VAL['d']=3
 OPT_NAME['p']='pulseaudio'   OPT_DESC['p']='disable pulseaudio on the test process'
 OPT_HAS_ARG['p']=0             OPT_VAL['p']=1
 
-func_opt_parse_option "$@"
+fxunc_opt_parse_option "$@"
+setup_kernel_check_point
```

Comment:
@fredoh9 @marc-hb maybe a stupid question, but there was a similar change from @aiChaoSONG for pm_runtime tests yesterday, so should we add this in ALL tests?

Reply:
This is more or less what I asked in https://github.com/thesofproject/sof-test/pull/639#issuecomment-818270484

Some (future?) tests may want to scan the entire log since boot though. For instance we discussed one "empty" test only for the purpose of checking SOF modules loaded at boot time. It's still not clear what such a test should do wrt checkpoints, @aiChaoSONG 

After excluding aliases and non-audio tests I found only 2 tests that don't invoke `setup_kernel_check_point()` directly:

```
+test-case/check-audio-equalizer.sh
+test-case/check-volume-levels.sh
```

Would these two tests also timeout if they fail?

Reply:
@plbossart @marc-hb Previously, when we were using DMESG_KERNEL_LAST_LINE which acted like kernel check point, this variable is implicitly set up in lib.sh. But I think it is better to set up check point explicitly in every test case, this will tell where the log collect is started.

Reply:
>so should we add this in ALL tests

I have a round of clean up on this within the journalctl PR, only minor number of test case(s) which is not run in PR test or daily test may need this. And also, when this happens now, some boundary condition is triggered.